pkg/doc/verifiable/presentation.go

```diff
@@ -178,49 +179,57 @@ func (vp *Presentation) JWTClaims(audience []string, minimizeVP bool) *JWTPresCl
 	return newJWTPresClaims(vp, audience, minimizeVP)
 }
 
-// Credentials provides Verifiable Credentials enclosed into Presentation in raw byte array format.
-func (vp *Presentation) Credentials() ([]PresentationCredential, error) {
-	marshalSingleCredFn := func(cred interface{}) (PresentationCredential, error) {
-		credBytes, err := json.Marshal(cred)
-		if err != nil {
-			return nil, fmt.Errorf("marshal credentials from presentation: %w", err)
-		}
+// Credentials returns current credentials of presentation.
+func (vp *Presentation) Credentials() []interface{} {
+	return vp.credentials
+}
 
-		return credBytes, nil
+// SetCredentials defines credentials of presentation. It could be Credential struct or encoded form (e.g. JWT).
```

Comment:
You can edit the comment saying cred can be either string/byte (probably serialized JWT) or Credential structure. It will just made it more readable. Just the minor comment

Reply:
Done!